operating/view-dashboard.adoc

```diff
@@ -6,58 +7,100 @@ include::modules/common-attributes.adoc[]
-The *Dashboard* view helps you to understand your cloud-native infrastructure, including images, containers, pods, namespaces, clusters, and their configurations.
+The {product-title} ({product-title-short}) dashboard enables you to visually track and analyze key metrics such as levels of risk in your environment, compliance status, and policy violations. The dashboard helps you to understand your cloud-native infrastructure, including images, containers, pods, namespaces, clusters, and their configurations.
 
 [NOTE]
 ====
-When you open the RHACS portal for the first time, you might see a blank dashboard.
-After you deploy Sensor in at least one cluster, the dashboard reflects the status of your environment.
+When you open the {product-title-short} portal for the first time, you might see a blank dashboard. After you deploy Sensor in at least one cluster, the dashboard reflects the status of your environment. 
 ====
 
-The *Dashboard* view includes multiple interactive widgets which show a summary of system violations, summary of compliance status, violations by cluster, top risky deployments, and active violations.
-It also includes other widgets that are automatically generated when you create new policy categories.
+The dashboard includes interactive widgets that provide different key performance indicators (KPIs) for violations, risks, and compliance. 
 //TODO: Add link to policy categories
 
-[id="system-violations"]
-== System Violations
+[id="overview-data"]
 
-The *System Violations* widget (upper-left) shows the number of critical, high, medium, and low severity violations on the *Dashboard* view.
-You can select a number on the widget to view details about those violations.
+The top of the dashboard shows an overview of system data, including the number of clusters, nodes, violations, deployments, images, and secrets. Choosing one of those items takes you to the main menu page for the item. For example, select *Deployments* to view the *Configuration Management* -> *Deployments* page.
 
-[id="compliance"]
-== Compliance
+[id="select-resources"]
+== Selecting resources for data display
 
-The *Compliance* widget (upper-right) shows the status of various compliance benchmarks and if their controls are passing or failing.
+You can review metrics for your entire system or only selected resources by selecting the following options:
 
-[NOTE]
-====
-The *Compliance* widget show details only after you run a compliance scan.
-====
-//TODO: Add link to compliance scan
+* To view data for specific clusters, select the clusters from the *Resources* drop-down list. 
+* To view data for all clusters, select *All clusters*. 
+* To view data for specific namespaces, select them from the *Resources* drop-down list. Note that all namespaces are automatically included when you select *All clusters*. 
```

Comment:
Yes this is correct, selecting a cluster will select all namespaces that are part of that cluster.

When the user goes from selecting "All clusters" to selecting a single cluster, nothing happens in the Namespace dropdown ("All namespaces" remains selected). If the user then selects another cluster, nothing happens still because the default behavior is "All namespaces".

The situation where this changes slightly is the following:
- The user selects a single cluster from the first dropdown. "All namespaces" remains selected in the second dropdown.
- The user then selects a subset of namespaces belonging to this cluster in the second dropdown.
<img width="657" alt="image" src="https://user-images.githubusercontent.com/1292638/178569316-ba6693c2-8234-40e3-88e3-00e04c55af82.png">
- The user then goes back and selects a second cluster from the first dropdown. The behavior remains the same, all namespaces from this second cluster are selected. However the UI is a little different, in that "All namespaces" will not be selected; instead all namespaces for the second cluster are automatically checked while the selection from the first cluster remains the same (a mix of checked/unchecked).
<img width="657" alt="image" src="https://user-images.githubusercontent.com/1292638/178569798-f56a1ded-56e9-4d28-848f-9d3d60dff042.png">

Maybe a more accurate wording could be "Note that selecting a cluster will automatically include all namespaces that are a part of that cluster".